sources/libertinussans-bold.sfd

```diff
@@ -8099,7 +8099,7 @@ SplineSet
  165 647 138 672 138 703 c 0
 EndSplineSet
 Refer: 24 65 N 1 0 0 1 0 0 2
-Position2: "'cpsp' Versalabstaende 1" dx=2 dy=0 dh=5 dv=0
+Position2: "'cpsp' Versalabstaende 1" dx=2 dy=0 dh=5 dv=0 [ddx={} ddy={} ddh={} ddv={}]
```

Comment:
Those empty ` [ddx={} ddy={} ddh={} ddv={}]` additions are just adding noise, please remove them manually for now (I’ll see if I can update the normalizer to handle them).